mozetl/taar/taar_locale.py

```diff
@@ -37,6 +40,26 @@ def store_new_state(source_file_name, s3_dest_file_name, s3_prefix, bucket):
     transfer.upload_file(source_file_name, bucket, key_path)
 
 
+def load_amo_external_whitelist():
+    white_list_inner = []
+    # Load the most current AMO dump JSON resource.
+    try:
+        s3 = boto3.client('s3')
+        s3_contents = s3.get_object(Bucket=AMO_DUMP_BUCKET, Key=AMO_DUMP_KEY)
+        amo_dump = json.loads(s3_contents['Body'].read())
```

Comment:
Let's define `amo_dump = {}` after line 44, this should make the test pass.

Reply:
sorry about this.